.github/actions/build/action.yml

```diff
@@ -1,6 +1,9 @@
 name: 'build'
 description: 'Build application from Dockerfile'
 inputs:
+  app_name:
+    required: true
+    description: "Application name based on which docker repository, doppler project and kube namespace would be selected"
```

Comment:
Can change the description here to - `Application name based on which build is cached`